lib/bolt/cli.rb

```diff
@@ -181,9 +181,13 @@ def create_option_parser(results)
                 "Output format to use: human or json") do |format|
           results[:format] = format
         end
-        opts.on('-k', '--insecure',
-                "Whether to connect insecurely ") do |insecure|
-          results[:insecure] = insecure
+        opts.on('--[no-]host-key-check',
+                "Check host keys with SSH") do |host_key_check|
+          results[:host_key_check] = host_key_check
+        end
+        opts.on('--[no-]ssl',
+                "Use ssl with WINRM") do |ssl|
```

Comment:
`SSL` should be capitalized, and it should be `WinRM`

Reply:
Fixed